spec/graphql/mutations/customers/update_spec.rb

```diff
@@ -59,6 +63,8 @@
       expect(result_data['invoiceGracePeriod']).to be_nil
       expect(result_data['providerCustomer']['id']).to be_present
       expect(result_data['providerCustomer']['providerCustomerId']).to eq('cu_12345')
+      expect(result_data['billingConfiguration']['documentLocale']).to eq('fr')
+      expect(result_data['billingConfiguration']['id']).to be
```

Comment:
RSpec/Be: Don't use `be` without an argument. (https://rspec.rubystyle.guide/#be-matcher, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Be)